modules/pagerduty-service/outputs.tf

```diff
@@ -2,3 +2,7 @@ output "id" {
   description = "The `id` attribute of the service."
   value       = pagerduty_service.service.id
 }
+
+output "service_name" {
```

Comment:
please validate this by adding to https://github.com/honestbank/terraform-pagerduty-alerting/blob/main/test/pagerduty_service_test.go